widget_driver_test/test/driver_tester_test.dart

```diff
@@ -1,7 +1,45 @@
-import 'package:test/scaffolding.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:widget_driver/widget_driver.dart';
+import 'package:widget_driver_test/src/driver_tester.dart';
+
+class MockWidgetDriver extends WidgetDriver {
+  void triggerNotifyWidgets() {
+    notifyWidget();
+  }
+}
 
 void main() {
-  test('sample test', () {
-    // TODO add tests
+  late MockWidgetDriver mockDriver;
```

Comment:
To make the printed output from the test a bit more readable, 
what do you think about wrapping all the tests in here inside a ´group('DriverTester:', () {})`?

This way the test report will print:

```
DriverTester: should wait for notifyWidget call
DriverTester: should verify no more calls to notifyWidget
```

You can have a look at [drivable_widget_test.dart](https://github.com/bmw-tech/widget_driver/blob/master/widget_driver/test/drivable_widget_test.dart) for some inspiration

So it would look something like:

```
void main() {
    group('DriverTester:', () {
        // All your existing code in here
    }
}
```

Reply:
Got it, thanks Robert. Update on commit a8dfb82b6772f553c3f36c02c6877e9e9a0c89f0

Reply:
I also created groups on f37f76dea0b947fceff89905890358182c144ff6 and 43039a5f1d8f518b8ac75ca5ee8907d9cc4e6ef6